### PR TITLE
Font loading and text spacing

### PR DESCRIPTION
**Pull request by allytart**

### CODE REVIEW

**Comment by allytart** [comment]
Minor changes to test font loading. Just need it merged so I can see if fonts load properly on mobile.

File: clinicians.html [hunk 1]
```diff
@@ -7,6 +7,8 @@
     <title>Clinicians | Fusiform</title>
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0" />
     <link rel="shortcut icon" href="img/fusiform/Brand_MainLogo_Circle.png" type="image/png">
+    <link href="fonts/avenir.css" rel="stylesheet" type="text/css" media="all" />
```
Comment (alexjmathews) [inline clinicians.html]: No importing avenir because its illegal LOL

File: clinicians.html [hunk 2]
```diff
@@ -7,6 +7,8 @@
...
+    <link href='https://fonts.googleapis.com/css?family=Lato:300,400%7CRaleway:100,400,300,500,600,700%7COpen+Sans:400,500,600' rel='stylesheet' type='text/css'>
```
Comment (alexjmathews) [inline clinicians.html]: Use webfont loader instead.

File: clinicians.html [hunk 3]
```diff
@@ -97,8 +97,8 @@
                 <span id="sec4">
                     <h8 class="labeltag bgTB colorF">CLINICIANS</h8>
                 </span>
-                <div style="margin: 0 0 0 30px;">
-                    <div class="container pt40 pb75">
+                <div>
```
Comment (alexjmathews) [inline clinicians.html]: Can we take out the comments?

File: press.html [hunk 4]
```diff
@@ -7,6 +7,8 @@
     <title>Press &amp; News | Fusiform</title>
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0" />
     <link rel="shortcut icon" href="img/fusiform/Brand_MainLogo_Circle.png" type="image/png">
+    <link href="fonts/avenir.css" rel="stylesheet" type="text/css" media="all" />
```
Comment (alexjmathews) [inline press.html]: same

File: clinicians.html [hunk 5]
```diff
@@ -15,8 +17,6 @@
     <link href="css/theme-nearblack.css" rel="stylesheet" type="text/css" media="all" />
     <link href="css/custom.css" rel="stylesheet" type="text/css" media="all" />
     <link href="css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all" />
```
Comment (alexjmathews) [inline clinicians.html:26]: This is 404-ing

File: index.html [hunk 6]
```diff
@@ -7,6 +7,8 @@
     <title>Fusiform - Fully Digital O&amp;P</title>
```
Comment (alexjmathews) [inline index.html:7]: faster transition for opening animation

**Review by alexjmathews — CHANGES_REQUESTED** [review]

File: index.html [hunk 7]
```diff
@@ -108,34 +107,34 @@ <h2 style="font-family: 'Poppins', sans-serif;">
...
+                            <h5 class="mb-xs-16 colorF">Cost containment keeps prices at a minimum while demand from your clinicians keeps increasing.</h5>
+                            <h5 class="mb-xs-16 colorF">How do you keep your O&amp;P business profitable? The only way is to increase <strong>operational efficiency</strong>, but O&amp;P is bogged down by inefficient practices like manual device ordering and fitting.</h5>
                         </div>
                         <!------START PINK BOX RIGHT------->
                         <div class="col-md-6 col-sm-5 col-md-push-1 col-sm-pull-1">
                             <div class="container mt30 accentboxP">
                                 <ul>
                                     <li>
-                                        <h4 class="colorLR">excessive inefficiencies</h4>
+                                        <h4 class="colorLR inconsolata">excessive inefficiencies</h4>
```
Comment (alexjmathews) [inline index.html:136]: @allytart This is running off the page. 

[https://www.dropbox.com/s/2yum8vq82phut3f/Screenshot%25202017-02-02%252018.07.50.png?dl=0](url)

**Review by alexjmathews — CHANGES_REQUESTED** [review]

File: index.html (hunk 7 above)
Comment (allytart) [reply index.html:136]: Can't see the attached image :(

**Review by alexjmathews — APPROVED** [review]

**Pull request merged**